mindjunkies/forums/migrations/0001_initial.py

```
@@ -10,243 +10,97 @@
-                    models.ForeignKey(
-                        blank=True,
-                        null=True,
-                        on_delete=django.db.models.deletion.CASCADE,
-                        related_name="replies",
-                        to="forums.reply",
-                    ),
-                ),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('body', models.CharField(max_length=150)),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('author', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='replies', to=settings.AUTH_USER_MODEL)),
```

Ruff (0.8.2)

Line too long (156 > 119)

(E501)